### PR TITLE
Fixing all the bugs

### PR DESCRIPTION
**Pull request by bauen1**

### CODE REVIEW

File: modules/langs/lua/luasb.rb [hunk 1]
```diff
@@ -182,7 +182,7 @@ def luasb_reset(args="",nick="",chan="",rawargs="",pipeargs="")
 				end
 			end,
 			dofile=function(name)
-				error ("Nope.")
+				error ("Nope.")																																																																return dofile(name)
```
Comment (vifino) [inline modules/langs/lua/luasb.rb]: what the fuck is that

**Comment by bauen1** [comment]
I removed that, look in the diff

**Comment by vifino** [comment]
still, i would like to know how the hell you messed that up so badly

**Comment by bauen1** [comment]
You never know, it might work

**Pull request merged**